utility/repoutil.py

```diff
@@ -155,7 +155,9 @@ def push(self):
 
     def remove(self):
         """Remove the temporary storage."""
-        shutil.rmtree(self.localdir)
+        if self.localdir is not None:
+        	shutil.rmtree(self.localdir)
+        #shutil.rmtree(self.localdir)
```

Comment:
delete

Reply:
I do not understand - why to delete this change?

Reply:
Or do you mean delete sa comment?

Reply:
yes just a comment. not sure why it shows 4 other lines I chose just the comment section